app/locales/sk.json

```diff
@@ -0,0 +1,112 @@
+{
+  "label": {
+    "about_title":"O",
```

Comment:
This look incorrect.

Reply:
I don't think so. This word was around the last time too. i had checked this. O is about in slovak :)

Reply:
will check with another native speaker too just to confirm.

Reply:
! cool. I learn :)